packages/launcher/__snapshots__/spec.ts.js

```diff
@@ -0,0 +1,128 @@
+exports['windows browser detection detects browsers as expected 1'] = [
+  {
+    "name": "chrome",
+    "family": "chromium",
+    "channel": "stable",
+    "displayName": "Chrome",
+    "versionRegex": {},
```

Comment:
where is major version?

Reply:
that gets added later on down the line, in the generic detection code, which is tested elsewhere